substrate/frame/nomination-pools/src/tests.rs

```diff
@@ -3290,8 +3301,9 @@ mod withdraw_unbonded {
 					unbond_pool.balance /= 2; // 295
 					SubPoolsStorage::<Runtime>::insert(1, sub_pools);
 					// Update the equivalent of the unbonding chunks for the `StakingMock`
-					let mut x = UnbondingBalanceMap::get();
-					*x.get_mut(&default_bonded_account()).unwrap() /= 5;
+					let x = UnbondingBalanceMap::get();
+					// TODO:
```

Comment:
👀

Reply:
I removed this test, as there was no reason to introduce it. It didn't exist before this PR.